devtools/create_project/xcode.cpp

```diff
@@ -555,6 +556,7 @@ void XcodeProvider::setupFrameworksBuildPhase(const BuildSetup &setup) {
 	frameworks_iOS.push_back("CoreGraphics.framework");
 	frameworks_iOS.push_back("CoreFoundation.framework");
 	frameworks_iOS.push_back("Foundation.framework");
+	frameworks_iOS.push_back("GameController.framework");
```

Comment:
`configure` and `ports.mk` also need to be updated for those not using Xcode to build the package (for example on our buildbot).